docs/en/latest/plugin-develop.md

```diff
@@ -103,7 +103,7 @@ local _M = {
 }
 ```
 
-Note : The priority of the new plugin cannot be the same as the priority of any existing plugin. In addition, plugins with a high priority value will be executed first in a given phase (see the definition of `phase` in [choose-phase-to-run](#choose-phase-to-run)). For example, the priority of example-plugin is 0 and the priority of ip-restriction is 3000. Therefore, the ip-restriction plugin will be executed first, then the example-plugin plugin.
+Note : The priority of the new plugin cannot be the same as the priority of any existing plugin, you can view the priority of all plugins in the [t/debug/debug-mode.t](../../../t/debug/debug-mode.t) file. In addition, plugins with a high priority value will be executed first in a given phase (see the definition of `phase` in [choose-phase-to-run](#choose-phase-to-run)). For example, the priority of example-plugin is 0 and the priority of ip-restriction is 3000. Therefore, the ip-restriction plugin will be executed first, then the example-plugin plugin.
```

Comment:
I think it would be better to use control API: https://github.com/apache/apisix/blob/master/docs/en/latest/control-api.md#get-v1schema

Reply:
I think that `Control API` has obtained the schema data of all plugins, and the content information is too much, and it is not intuitive to check the priority of all plugins.

Reply:
No, it is the official way to check the priority since users may add their own plugins and those plugins can't be reflected in our test file.

Reply:
Okay, but I think there is too much data obtained through `/v1/schema`. We should add a new API interface, such as `/v1/plugins/priority`, this API only returns the priority value of all plugins, and they are ordered. What do you think?

Reply:
No. Don't make thing complex. The method is called rarely.

Reply:
got it.